src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,18 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
+  isProjectSettingsPage = false,
```

Comment:
What is the purpose of this prop? Would it be possible for the ternary below to only check if a repoIdentifier/optionGroupLink exists?

Reply:
The headers should be clickable on the project settings page, but shouldn't be clickable on the waterfall. I'll add a comment to make it more readable.

Reply:
Since the waterfall doesn't provide a repoIdentifier, would it not be sufficient to check if one exists?

Reply:
It feels too risky to rely on that because it's just accessing the repoIdentifier through the project ref. Right now GET_PROJECTS which is how the waterfall page fetches projects doesn't get the repoRefId, but that may change down the line and whoever makes the change will have no reason to suspect that adding it will completely change the project dropdown.